packages/ui/cypress/e2e/login/index.spec.js

```diff
@@ -0,0 +1,103 @@
+describe('Login', () => {
+  context('with login form', () => {
+    beforeEach(() => {
+      cy.visit('/login')
+    })
+
+    it('can login by pressing enter', () => {
+      cy.get('[name=username]')
+        .type('test')
+        .get('[name=password]')
+        .type('password{enter}')
+        .url()
+        .should('include', '/app')
+    })
+
+    it('can login by clicking login button', () => {
+      cy.get('[name=username]')
+        .type('test')
+        .get('[name=password]')
+        .type('password')
+        .get('[name=btn-login]')
+        .click()
+        .url()
+        .should('include', '/app')
+    })
+
+    it('disable login button if username and password are empty', () => {
+      cy.get('[name=btn-login]').should('be.disabled')
+    })
+
+    it('shows an validation error if username is empty', () => {
+      cy.get('[name=password]')
+        .type('test')
+        .get('[name=btn-login]')
+        .click()
+        .getByText('Required')
+        .url()
+        .should('not.include', '/app')
+    })
+
+    it('shows an validation error if password is empty', () => {
+      cy.get('[name=username]')
+        .type('test')
+        .get('[name=btn-login]')
+        .click()
+        .getByText('Required')
+        .url()
+        .should('not.include', '/app')
+    })
+
+    it('shows an validation error if usrname is removed', () => {
+      cy.get('[name=username]')
+        .type('test{selectall}{backspace}')
+        .blur()
+        .getByText('Required')
```

Comment:
how about adding .should('be.visibile') to this? so it becomes an assertion in the test protocol?